py-polars/src/error.rs

```diff
@@ -48,6 +48,7 @@ impl std::convert::From<PyPolarsErr> for PyErr {
                 PolarsError::StructFieldNotFound(name) => {
                     StructFieldNotFoundError::new_err(name.to_string())
                 },
+                PolarsError::FileNotFound(err) => FileNotFoundError::new_err(err.to_string()),
```

Comment:
This should map to a [PyFileNotFoundError](https://docs.rs/pyo3/latest/pyo3/exceptions/struct.PyFileNotFoundError.html) (which you'll have to import from PyO3):

```suggestion
                PolarsError::FileNotFound(err) => PyFileNotFoundError::new_err(err.to_string()),
```

Reply:
Done! So that I understand it for next time: what is the advantage/difference of having a 'native' python error over the one i implemented initially?

Reply:
It better integrates with existing error handling in Python code.

If we raise a custom error, people need to import and specifically handle that custom error.